Command/GenerateTokenCommand.php

```diff
@@ -2,6 +2,9 @@
 
 namespace Lexik\Bundle\JWTAuthenticationBundle\Command;
 
+use Traversable;
+use Countable;
+use RuntimeException;
```

Comment:
same CS as Symfony here :) I don't like having use statements for things coming from the global namespace

Reply:
I understand. I removed the rule and reverted the changes